circuits/cpp/barretenberg/cpp/src/barretenberg/common/fuzzer.hpp

```diff
@@ -591,12 +597,14 @@ class ArithmeticFuzzHelper {
      * @param instructions
      */
     template <typename Composer>
-    inline static void executeInstructions(std::vector<typename T::Instruction>& instructions)
-        requires CheckableComposer<Composer>
+    // TODO(@Rumata888)(from Zac: maybe try to fix? I'm not comfortable refactoring this myself!)
```

Comment:
Not sure about the policy for TODOs, but generally I'd create an issue for these since they get lost in the codebase

Reply:
added one